egs2/librispeech/asr1/inference_pretrained.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
```

Comment:
Where are you using this?
Is it required?

Reply:
I copy from the original PR, so I suppose that it is employed for tests. But I will remove them before PR.

Reply:
@julien-c, could you tell me where this script is used?